test_schemas/en/test_question_title_in_error.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
I little bit verbose, it could easily be one textfield with an error message

Reply:
Each field uses a field handler to do the formatting, while the mutually exclusive uses a different approach in the questionnaire form. This schema tests both those approaches.